src/content/lodash/function/after/spec.js

```diff
@@ -0,0 +1,24 @@
+const fns = {
+  lodash: require('./lodash'),
+  plain: require('./vanilla'),
+}
+
+describe('before', () => {
+  Object.entries(fns).forEach(([key, method]) => {
+    describe(key, () => {
+      let fn
+      beforeEach(() => {
+        fn = jest.fn(a => a)
+      })
+
+      test('it should call the fn only the first n-1 times and then return the result of the last operation', () => {
+        const befored = method(5, fn)
```

Comment:
test description and const name are also likely required before -> after fixes

Reply:
yep, damn copy/paste, I will fix them